src/identity-handler.js

```diff
@@ -21,7 +21,14 @@ For more userIdentity types, see https://docs.mparticle.com/developers/sdk/web/i
 function IdentityHandler(common) {
     this.common = common || {};
 }
-IdentityHandler.prototype.onUserIdentified = function(mParticleUser) {};
+IdentityHandler.prototype.onUserIdentified = function(mParticleUser) {
+    var identitiesObject = mParticleUser.getUserIdentities();
+    var identity = identitiesObject.userIdentities[this.common.userIdentificationType];
+
+    if (identity) {
+        window.heap.identify(identity);
+    }
+};
```

Comment:
I think we should be defensive in case the user or identities are empty.

```suggestion
    if (!mParticleUser && !mParticleUser.getUserIdentities()){
        return;
    }

    var identitiesObject = mParticleUser.getUserIdentities();
    var identity = identitiesObject.userIdentities[this.common.userIdentificationType];

    if (identity) {
        window.heap.identify(identity);
    }
};
```

Reply:
Updated.